packages/pattern-commands/src/lib/structures/PatternCommand.ts

```diff
@@ -1,4 +1,4 @@
-import { Args, Command } from '@sapphire/framework';
+import { Args, Command, MessageCommand } from '@sapphire/framework';
```

Comment:
`Command` is not used

Reply:
`PatternCommand` class is extended from `Command`